packages/kbn-migrations/src/lib/migration_context.js

```diff
@@ -0,0 +1,51 @@
+// The migration context object contains all of the data and functions
+// that are necessary to analyze and run migrations.
+
+const _ = require('lodash');
+const objectHash = require('object-hash');
+const Plugin = require('./plugin');
+const MigrationPlan = require('./migration_plan');
+const MigrationState = require('./migration_state');
+const Persistence = require('./persistence');
+
+module.exports = {
+  fetch,
+};
+
+async function fetch(opts) {
+  const { callCluster, log, index, plugins, elasticVersion, force } = opts;
+  const initialIndex = `${index}-${elasticVersion}-original`.toLowerCase();
```

Comment:
Why lowercase it?

Reply:
Elastic doesn't appear to like indices w/ caps in them: `Invalid index name [BAR], must be lowercase`.